server/util/twilio.js

```diff
@@ -2,16 +2,32 @@ import Twilio from 'twilio';
 
 // Account SID and Auth Token are stored as environmental variables
 // Twilio Node module will check for them automatically upon initialization
-//
-//
-//
-// UNCOMMENT THE BELOW TWO LINES WHEN MERGING THE PLACE CALLS PR
-// |
-// |
-// |
-// V
-// const twilioClient = new Twilio();
-// const CALLER_ID = process.env.TWILIO_CALLER_ID;
+const twilioClient = new Twilio();
+
+const CALLER_ID = process.env.TWILIO_CALLER_ID;
+const DEV_PATH = process.env.DEV_PATH;
+
+export function callVolunteer(user_id, campaign_id, phone_number) {
+  return new Promise((resolve, reject) => {
+    twilioClient.calls.create({
```

Comment:
It seems that this already returns a promise: https://twilio.github.io/twilio-node/3.6.3/Twilio.Api.V2010.AccountContext.CallList.html

Do you think we can get rid of the extraneous promise wrapper you put in place and just return what the client returns for this function call?